engine/tests/story/test_choice_availability.py

```
@@ -103,6 +103,15 @@ def test_satisfied_predicate_makes_choice_available(self) -> None:
 
         assert fragments and fragments[0].available is True
 
+    def test_satisfied_action_predicate_makes_choice_available(self) -> None:
+        graph, start, _end, action = _graph_with_choice()
```

Ruff (0.15.12)

[warning] Unpacked variable `graph` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)